opencga-core/src/main/java/org/opencb/opencga/core/tools/result/ExecutionResultManager.java

```diff
@@ -312,7 +312,7 @@ public interface ExecutionResultFunction<R> {
         R apply(ExecutionResult execution) throws ToolException;
     }
 
-    private synchronized <R> R updateResult(ExecutionResultFunction<R> update) throws ToolException {
+    public synchronized <R> R updateResult(ExecutionResultFunction<R> update) throws ToolException {
```

Comment:
This method was left "private" on purpose, so all updates are controlled through the specific API methods.